docs/releases/5.1.md

```diff
@@ -122,6 +122,59 @@ The undocumented `users_with_page_permission` function in `wagtail.admin.auth` i
 
 ## `GroupPagePermission` now uses Django's `Permission` model
 
-The `GroupPagePermission` model that is responsible for assigning page permissions to groups now uses Django's `Permission` model instead of a custom string. This means that the `permission_type` `CharField` has been replaced with a `permission` `ForeignKey` to the `Permission` model.
+The `GroupPagePermission` model that is responsible for assigning page permissions to groups now uses Django's `Permission` model instead of a custom string. This means that the `permission_type` `CharField` has been deprecated and replaced with a `permission` `ForeignKey` to the `Permission` model.
 
 In addition to this, "edit" permissions now use the term `change` within the code. As a result, `GroupPagePermission`s that were previously recorded with `permission_type="edit"` are now recorded with a `Permission` object that has the `codename="change_page"` and a `content_type` that points to the `Page` model. Any permission checks that are done using `PagePermissionPolicy` should also use `change` instead of `edit`.
+
+If you have any fixtures for the `GroupPagePermission` model, you will need to update them to use the new `Permission` model. For example, if you have a fixture that looks like this:
+
+```json
+{
+  "pk": 11,
+  "model": "wagtailcore.grouppagepermission",
+  "fields": {
+    "group": ["Event moderators"],
+    "page": 12,
+    "permission_type": "edit"
+  }
+}
+```
+
+Update it to use a natural key for the `permission` field instead of the `permission_type` field:
+
+```json
+{
+  "pk": 11,
+  "model": "wagtailcore.grouppagepermission",
+  "fields": {
+    "group": ["Event moderators"],
+    "page": 12,
+    "permission": ["change_page", "wagtailcore", "page"]
+  }
+}
+```
+
+If you have any code that creates `GroupPagePermission` objects, you will need to update it to use the `Permission` model instead of the `permission_type` string. For example, if you have code that looks like this:
+
+```python
+from wagtail.core.models import GroupPagePermission
+
+permission = GroupPagePermission(group=group, page=page, permission_type="edit")
+permission.save()
```

Comment:
We could also mention that the `GroupPagePermission.objects.create(group=group, page=page, permission_type="change")` shorthand still exists and won't be removed in 6.0, but I decided not to as I don't want people to depend on it in case we remove it. I'm merely keeping the shorthand so that we don't have to update existing tests too much.